src/main/scala/io/digitallibrary/bookapi/model/domain/PdfCss.scala

```diff
@@ -7,44 +7,119 @@
 
 package io.digitallibrary.bookapi.model.domain
 
-case class PdfCss (source: Option[String], fontNames: Seq[String]) {
-
-  val DefaultPageSize = "a4"
-  val landscapeSources = Map(
-    "storyweaver" -> "a5 landscape",
-    "taf" -> "a5 landscape",
-    "bookdash" -> "a5 landscape"
-  )
+case class PdfCss (source: Option[String], pageOrientation: PageOrientation.Value, fontNames: Seq[String]) {
 
   def asString: String = {
+    pageOrientation match {
+      case PageOrientation.PORTRAIT => portraitCss
+      case PageOrientation.LANDSCAPE => landscapeCss
+    }
+  }
+
+  def landscapeCss: String = {
```

Comment:
Det er såvidt jeg kan se kun 4 forskjeller i de resulterende css-strengene for landscape og portrait, altså mye likt. Hadde det vært lettere å vedlikeholde om dette ble refaktorert til 1 funksjon som kun skiller på typen disse 4 stedene? Noe ala dette:
```scala
def css: String = {
   val pageSize = pageOrientation match {
      case PageOrientation.PORTRAIT => "a5 landscape"
      case PageOrientation.LANDSCAPE => "a5"
    }
    
    val imgMaxWidth = pageOrientation match {
      case PageOrientation.PORTRAIT => "400px"
      case PageOrientation.LANDSCAPE => "500px"
    }

   (…)
    
    s"""
      |@page {
      |   size: $pageSize;
      |}
(…)
```
Da mener jeg det blir mye tydeligere hva som er forskjellene, og vi kan holde de øvrige verdiene like for begge tilfeller.